grammar/Makefile

```diff
@@ -0,0 +1,9 @@
+ANTLR_JAR=antlr-4.13.2-complete.jar
+GRAMMARS=SubstraitLexer.g4 SubstraitType.g4
+OUTPUT_DIR=../parser/types/baseparser
+
+generate:
```

Comment:
we should be download grammar from substrait core rather than checking into this repo.

Reply:
Waiting on this [grammar PR](https://github.com/substrait-io/substrait/pull/730)  to get merged

Reply:
Done